<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SF_Widget</class>
 <widget class="QWidget" name="SF_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="mnemoscheme">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>450</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Russia"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="pixmap">
    <pixmap>mnemo/тб.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="temp_lcd">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>290</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="value" stdset="0">
    <double>123.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>123</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pres_lcd">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>290</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton
{
	color: red;
}
QRadioButton:indicator:checked
{
	image: url(:/Mnemoscheme/icons/greenlamp.svg);
	width: 15px;
	height: 15px;
}
QRadioButton:indicator:unchecked
{
	image: url(:/Mnemoscheme/icons/redlamp.svg);
	width: 15px;
	height: 15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="value" stdset="0">
    <double>123.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>123</number>
   </property>
  </widget>
  <widget class="QLabel" name="he_pump_led">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>294</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="1234.qrc">:/Mnemoscheme/icons/redlamp.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="he_input_led">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>249</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="1234.qrc">:/Mnemoscheme/icons/redlamp.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="he_output_led">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>294</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="1234.qrc">:/Mnemoscheme/icons/redlamp.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="high_level_led">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>236</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="1234.qrc">:/Mnemoscheme/icons/redlamp.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="output_valve_led">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>188</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="1234.qrc">:/Mnemoscheme/icons/redlamp.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="output_pump_led">
   <property name="geometry">
    <rect>
     <x>547</x>
     <y>188</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="1234.qrc">:/Mnemoscheme/icons/redlamp.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="co2_valve_led">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>118</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="1234.qrc">:/Mnemoscheme/icons/redlamp.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="low_level_led">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>358</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="1234.qrc">:/Mnemoscheme/icons/redlamp.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="input_valve_led">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>188</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="1234.qrc">:/Mnemoscheme/icons/redlamp.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="1234.qrc"/>
  <include location="1234.qrc"/>
 </resources>
 <connections/>
</ui>
